tests/hardhat/Fork/RewardsForkTest.ts

```diff
@@ -0,0 +1,265 @@
+import { smock } from "@defi-wonderland/smock";
+import { mine } from "@nomicfoundation/hardhat-network-helpers";
+import chai from "chai";
+import { BigNumberish, Signer } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { convertToUnit } from "../../../helpers/utils";
+import {
+  AccessControlManager,
+  AccessControlManager__factory,
+  Comptroller,
+  Comptroller__factory,
+  MockToken,
+  MockToken__factory,
+  RewardsDistributor,
+  RewardsDistributor__factory,
+  VToken,
+  VToken__factory,
+} from "../../../typechain";
+import { initMainnetUser, setForkBlock } from "./utils";
+
+const { expect } = chai;
+chai.use(smock.matchers);
+
+const FORK_TESTNET = process.env.FORK_TESTNET === "true";
+const FORK_MAINNET = process.env.FORK_MAINNET === "true";
+const MANTISSA_ONE = convertToUnit(1, 18);
+
+let ADMIN: string;
+let ORACLE_ADMIN: string;
```

Comment:
We don't use it

Reply:
Fixed [here](https://github.com/VenusProtocol/isolated-pools/pull/236/commits/3c4f565094eaf1e27db7e5a20156f7f812a427f8)